provision/acc_provision/acc_provision.py

```diff
@@ -123,7 +123,7 @@ def config_default():
     # Default values for configuration
     default_config = {
         "aci_config": {
-            "apic_version": 1.0,
+            "apic_version": "1.0",
```

Comment:
Instead of setting the apic_version as "a.b.c", can we read it in the actual format "a.b(cz)" (e.g. "5.2(3e)") which you expect to get from the APIC?

Reply:
This field aci_config.apic_version will always look like a.b.c since we process the incoming version from APIC

Reply:
My understanding is that we are using apic_version primarily for UT purposes. So why can we not interpret it in the format that we expect from APIC? That helps us to test the real APIC format even in UTs, no?

Reply:
Based off our offline discussion, created an issue for this.